gdsc-web-backend/Controllers/SettingsController.cs

```diff
@@ -43,5 +43,26 @@ public SettingModel Get(string id)
         {
             return Settings.Find(x => x.Id == id);
         }
+
+        [HttpPost]
+        public List<SettingModel> Post(SettingModel setting)
```

Comment:
BadRequest or Created or NotFound are action results. This means that you should wrap your returning type with an ActionResult<> like `ActionResult<List<SettingModel>>`. 
Please run you application before commit(`dotnet watch run`). It will help you find errors like these